package.json

```diff
@@ -8,11 +8,17 @@
     "is-valid-domain": "^0.1.6"
   },
   "devDependencies": {
+    "@types/jest": "^27.5.1",
```

Comment:
I gives us a nice intellisense though :)

Reply:
I think you can add it as a global package to your vscode if you want the intellisense (I've done that in the past but it might have been in webstorm and not vscode), otherwise I'd say it's unnecessarily included in the resulting image

Reply:
On that note, we should probably run `yarn --prod` in our `Dockerfile`s so dev dependencies aren't be installed (I created [SKY-867](https://linear.app/skynetlabs/issue/SKY-867/run-yarn-in-production-mode)).

But fair enough, removed in [3f7dfd6](https://github.com/SkynetLabs/webportal-dnslink-api/pull/5/commits/3f7dfd6a54b14fc427e9cc53553c1bfd01d1b469)